.changeset/itchy-spiders-scream.md

```diff
@@ -0,0 +1,6 @@
+---
+'@openfn/language-salesforce': major
```

Comment:
I don't think we want this change to be a major version bump. Ideally this should be a patch. It make sense to be a major version bump because of breaking changes but the latest release is not used on any project yet. 

@josephjclark Do you have any opinions on this ?

Reply:
Agreed - I think minor is fine